src/Uno.UI/UI/Xaml/Input/Internal/ManipulationEventArgsHelpers.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Windows.Foundation;
+using Windows.UI.Xaml;
+
+namespace Uno.UI.UI.Xaml.Input.Internal
+{
+	internal static class ManipulationEventArgsHelpers
+	{
+		internal static Point MapPointRelativeTo(UIElement element, Point point)
+			=> element is null
+				? point
+				: element.TransformToVisual(null).Inverse.TransformPoint(point);
```

Comment:
Stolen from @Xiaoy312 sample :) Thanks for the workaround!

Reply:
I'm really not a fan of this one: `TransformToVisual(null)` is really intense as it required to walk the whole tree. Depending of the platform we can have an optimized way to get relative coordinates from the `PointerRoutedEventArgs` that are not so far from here ...

Reply:
Let me close the PR for now. I wasn't able to get it, and the workaround can easily be integrated into apps that need it. So no need for such a workaround in Uno itself.